low-level/sensors/impl/generic/node_hw.py

```diff
@@ -1187,6 +1284,15 @@ def _parse_temperature_info(self, index, date, _time, sensor, sensor_num, event,
             "description": event
         }
 
+        if (threshold.lower() in ['low', 'high'] and not status.lower == "deasserted"):
+            self.faulty_resources[sensor_name] = {
+                'status': status,
+                'event': event,
+                'resource': 'temperature'
```

Comment:
This should be corrected as mentioned for other fru types.
Please take care.

Reply:
Done